floodviz/static/js/map.js

```diff
@@ -66,14 +66,18 @@ document.addEventListener("DOMContentLoaded", function (event) {
 	add_circles(FV.mapinfo.ref_data, "ref-point", 2);
 	var sites = add_circles(FV.mapinfo.site_data, "gage-point", 3);
 	sites.selectAll("circle")
-		.on('mousemove', function (d) {
+		.on("mouseover", function (d) {
```

Comment:
I think it's better as an anonymous function because we may end up adding different mouseover actions to the other features on the map.